docs/docsite/rst/getting_started/get_started_inventory.rst

```diff
@@ -47,9 +49,11 @@ Continue getting started with Ansible by :ref:`creating a playbook<get_started_p
 Inventories in INI or YAML format
 =================================
 
-You can create inventories in either `INI` files or in YAML.
+You can create inventories in either `INI` files or in `YAML`.
```

Comment:
Shouldn't this use double backticks? Same below.
```suggestion
You can create inventories in either ``INI`` files or in ``YAML``.
```

Reply:
LGTM, committed, thanks! (i also changed other places similarly). PTAL